editor/src/utils/feature-switches.ts

```diff
@@ -70,7 +70,7 @@ let FeatureSwitches: { [feature in FeatureName]: boolean } = {
   'Condensed Navigator Entries': !IS_TEST_ENVIRONMENT,
   'Use Parsing Cache': false,
   'Canvas Fast Selection Hack': true,
-  'Import Wizard': false,
+  'Import Wizard': true,
```

Comment:
turning it on now that there is a way to continue import (there are no blocking states over the current implementation)